WebDriverAgentLib/Routing/FBElement.m

```diff
@@ -7,9 +7,21 @@
  * of patent rights can be found in the PATENTS file in the same directory.
  */
 
+#import <Foundation/Foundation.h>
+
 #import "FBElement.h"
+#import "FBElementTypeTransformer.h"
 
 NSString *wdAttributeNameForAttributeName(NSString *name)
 {
   return [NSString stringWithFormat:@"wd%@", name.capitalizedString];
 }
+
+NSSet<NSNumber *> *wdGetUniqueElementsTypes(NSArray<id<FBElement>> *elements)
```

Comment:
We should move both `wdAttributeNameForAttributeName` and `wdGetUniqueElementsTypes` to class methods of a new class maybe `FBElementUtils`?